c3/generator/devices.py

```diff
@@ -1,6 +1,5 @@
 import os
```

Comment:
All devices need to conform to the signal chain usage, i.e. implement `process()`. Confirm with tests.

Reply:
We'll do this in dev.